types/host.pp

```diff
@@ -1,2 +1,2 @@
 # @summary Validate a host (FQDN or IP address)
-type Stdlib::Host = Variant[Stdlib::Fqdn, Stdlib::Ip::Address]
+type Stdlib::Host = Variant[Stdlib::Fqdn, Stdlib::IP::Address]
```

Comment:
This now matches the casing here: https://github.com/puppetlabs/puppetlabs-stdlib/blob/883ba57ada149b32deff4cd1ef51de3bb51a9b30/types/ip/address.pp#L2

Reply:
Interesting :thinking:.  For some reason I stick to `\A[[:upper:]][[:lower:]]*\z` but it seems having more capital letters is valid…  I guess what annoyed me is that I wanted to camel_case the type name for the file name as we do with Ruby but this is not supported…